src/libs/java/gradle.properties

```diff
@@ -1 +1 @@
-syslVersion=0.2.8
+syslVersion=0.2.9
```

Comment:
Why 0.2.9 and not 0.3? Needs a justification in the PR description.

Reply:
Updated a list of features delivered in this release. We have delivered more tech debts/bug fixes this release than new features. In my opinion this is what makes it a minor release.